README.md

```diff
@@ -25,140 +26,115 @@ $ pip install mollie-api-python
+There we can use the `id` from our POST parameters to check te status and act upon that, see [Example 2 - Webhook verification](https://github.com/mollie/mollie-api-python/blob/master/examples/02-webhook-verification.py).
 
-```python
-issuers = mollie_client.issuers.all()
-```
-
-_`issuers` will be a list of `mollie.api.objects.Issuer` objects. Use the attribute `id` of this object in the
- API call, and the attribute `name` for displaying the issuer to your customer. For a more in-depth example, see [Example 4](https://github.com/mollie/mollie-api-python/blob/master/examples/4-ideal-payment.py)._
 
-Create a payment with the selected issuer:
+## Multicurrency ##
+Since 2.0 it is now possible to create non-EUR payments for your customers.
+A full list of available currencies can be found [in our documentation](https://docs.mollie.com/guides/multicurrency).
 
 ```python
 payment = mollie_client.payments.create({
-    'amount':      10.00,
-    'description': 'My first API payment',
+    'amount': {'currency': 'USD', 'value': '10.00'},
+    'description': 'Order #12345',
     'redirectUrl': 'https://webshop.example.org/order/12345/',
-    'webhookUrl':  'https://webshop.example.org/mollie-webhook/',
-    'method':      mollie.api.objects.Method.IDEAL,
-    'issuer':      selected_issuer_id,  # e.g. 'ideal_INGBNL2A'
+    'webhookUrl': 'https://webshop.example.org/mollie-webhook/',
 })
 ```
-
-_The `payment_url` attribute of the `payment` object will point directly to the online banking environment of the selected issuer._
+_After creation, the `settlementAmount` will contain the EUR amount that will be settled on your account._
 
 ### Refunding payments ###
 
 The API also supports refunding payments. Note that there is no confirmation and that all refunds are immediate and
-definitive. Refunds are only supported for iDEAL, credit card and Bank Transfer payments. Other types of payments cannot
-be refunded through our API at the moment.
+definitive. Refunds are only supported for iDEAL, credit card, Bancontact, SOFORT Banking, PayPal, Belfius Direct Net, KBC/CBC, 
+ING Home'Pay and bank transfer payments. Other types of payments cannot be refunded through our API at the moment.
 
 ```python
-payment = mollie_client.payments.get(payment['id'])
-refund = mollie_client.payments.refund(payment)
+payment = mollie_client.payment.get(payment.id)
+
+# Refund € 2 of this payment
+refund = payment.refund({
+    'amount': {
+        'currency': 'EUR',
+        'value': '2.00'
+    }
+})
 ```
 
-## Examples ##
+For a working example, see [Example 7 - Refund payment](https://github.com/mollie/mollie-api-python/blob/master/examples/07-refund-payment.py).
 
-To run the examples you need to install [Flask](http://flask.pocoo.org/). Simply run:
+## API documentation ##
+If you wish to learn more about our API, please visit the [Mollie Developer Portal](https://www.mollie.com/en/developers). API Documentation is available in English.
 
-```
-$ cd mollie-api-python
```

Comment:
dit moet er natuurlijk wel in. Nu weete niemand hoe dit werkt. Graag een variant maken die het  doet met een virtualenv, zodat die leleijke import workarounds die we hebben weggehaald in de example files niet terug hoeven.